summary_workflows/differential_usage/diff.usage.barplot.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import os, re, argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def bar_plot(files, plot_out):
+	'''
+	Inputs:
+		- files: a list of tsv files containing 
+			  	gene_id & usage changes (e.g. deltaPPAU in QAPA);
+			  	NOTE that these genes have been filtered by FDR/q-value.
+			  	File names are like 'AA_diffUTR_03.tsv'
+		- [hardcoded] cutoff: the cutoff to determine if the change is big enough or not,
+				  e.g. in QAPA, deltaPPAU > 20 means lengthening
+		- output: path to store the image
+	'''
+
+	files = [f for f in files if '_03.tsv' in f]
+	methods = {}
+	for file in files:
+		method = re.sub('_03.tsv$', '', os.path.basename(file))
+		method = re.sub('^.*_', '', method)
+
+		# hardcode the cutoffs since each method has its own calculation
+		cutoff = 0
+		if method == "QAPA":
+			cutoff = 20
+		elif method == "diffUTR":
+			cutoff = 0.5 #not sure, can ask the author
```

Comment:
I agree with the points @lschaerfen and @uniqueg mentioned. I personally prefer flexibility in specifying the cutoffs unless the authors specifically mention that a certain threshold is a gold standard, which requires referencing. Other than that, everything looks good to me!